doc-src/install.rst

```diff
@@ -293,10 +295,18 @@ The recommended download method is :code:`git clone --recursive`.
 Build
 -----
 
-To build, simply::
+To build, use::
 
+  $ ./configure
   $ make
 
+  ./confgure has several options that can be specified at build time
+  --enable-docs (Build the docs)
+  --enable-tests (Build the test suite)
+  --disable-docker (Disable docker support)
+  --disable-chgrow (Disable ch-grow support)
+  --disable-buildah (Disable buildah support) 
```

Comment:
The idea is to be able to short circuit parts of the helper script that don't need to be there if said builder is not installed for exampple. I'm not sure how this will work in practice.